libs/wingsdk/src/target-sim/function.inflight.ts

```diff
@@ -18,6 +21,8 @@ export class Function implements IFunctionClient, ISimulatorResourceInstance {
   private readonly context: ISimulatorContext;
   private readonly timeout: number;
 
+  private bundle?: string;
```

Comment:
Docstring would be nice here (name isn't self-descriptive)

Reply:
Done